packages/lib/src/model.ts

```diff
@@ -11,7 +11,9 @@ export enum ModelID {
   Claude_V1_100k = "anthropic/claude-v1-100k",
   // Other
   Together = "togethercomputer/GPT-NeoXT-Chat-Base-20B",
-  Cohere = "cohere/command-nightly"
+  Cohere = "cohere/command-nightly",
+  // Media Models
+  Shap_e = "openrouter/shap_e",
```

Comment:
Seems like this one should be called `"openai/shape-e"` for consistency

Reply:
I agree - changed!